modules/seqexec/server/src/test/scala/seqexec/server/TestEpicsCommand.scala

```diff
@@ -0,0 +1,106 @@
+package seqexec.server
+
+import cats.{Applicative, Monad}
+import cats.effect.concurrent.Ref
+import cats.implicits._
+import monocle.Lens
+import monocle.macros.Lenses
+import seqexec.model.enum.ApplyCommandResult
+import squants.Time
+
+
+object TestEpicsCommand {
+
```

Comment:
Partial implementation of `EpicsCommand` with up to 4 parameters. When run, the commands will append an event to the output `List`.